app/main.py

```diff
@@ -1 +1,47 @@
-# write your code here
+class SoftwareEngineer:
+
+    skills = []
+
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def learn_skill(self, skill: str) -> None:
+        if skill not in self.skills:
+            self.skills.extend([skill])
+
+
+class FrontendDeveloper(SoftwareEngineer):
+
```

Comment:
You should return the `__init__` method, and for every skill in skills use method`learn_skill()`. The same in `BackendDeveloper`